### PR TITLE
Added infusion dex on base chain

### PR DESCRIPTION
**Pull request by Jason4276**
Added trades data  for `infusion` On `Base`

### CODE REVIEW

File: dbt_subprojects/dex/models/trades/base/_schema.yml [hunk 1]
```diff
@@ -996,3 +996,20 @@ models:
             - evt_index
       - check_dex_base_trades_seed:
           seed_file: ref('xchange_base_base_trades_seed')
+
+  - name: infusion_base_base_trades
+    meta:
+      blockchain: base
+      sector: dex
+      project: infusion
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/base/_schema.yml:1004]: Please add `infusion` in `dex.info`:
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/dex/models/dex_info.sql

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @Jason4276 

Please run `git merge main` to update this PR as well.

[Sample infusion_base_base_trades](https://dune.com/queries/4367529)

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
@Jason4276 i've been doing merges that also add to base chain for DEX spells. this causes merge conflicts, but i don't have access to your branch to help resolve. plz do fix and push to the PR

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
@Jason4276 sorry, i had another base dex to merge, now we have another conflict 🫠

**Comment by Jason4276** [comment]
@jeff-dude It’s okay; I will resolve it.

**Pull request merged**